rosidl_typesupport_opensplice_cpp/bin/rosidl_typesupport_opensplice_cpp

```diff
@@ -8,6 +9,15 @@ from rosidl_typesupport_opensplice_cpp import generate_dds_opensplice_cpp
 from rosidl_typesupport_opensplice_cpp import generate_typesupport_opensplice_cpp
 
 
+def is_valid_file(parser, file_name):
+    if not os.path.exists(file_name):
+        parser.error("File does not exist: '{0}'".format(file_name))
+    file_name = os.path.abspath(file_name)
```

Comment:
The validation function might not want to change the argument to an absolute file. That is kind of unexpected.